src/components/radio/radio-button.scss

```diff
@@ -0,0 +1,133 @@
+@import "default-theme";
+
+$radio-width: 20px !default;
+$radio-height: $radio-width !default;
+
+md-radio-button {
+  display: inline-block;
+  cursor: pointer;
+  padding: 8px;
+  position: relative;
+  white-space: nowrap;
+}
+
+md-radio-group {
+}
+
+.md-radio-container {
+  box-sizing: border-box;
+  display: inline-block;
+  height: $radio-width;
+  position: relative;
+  top: 2px;
+  width: $radio-width;
+}
+
+// TODO(mtlin): Replace when ink ripple component is implemented.
+.md-ink-ripple {
+  background: #000;
+  border-radius: 50%;
+  height: 48px;
+  left: 10px;
+  opacity: 0;
+  pointer-events: none;
+  position: absolute;
+  top: 10px;
+  transform: translate(-50%,-50%);
+  transition: opacity ease 0.28s, background-color ease 0.28s;
+  width: 48px;
+  overflow: hidden;
+}
+
+.md-radio-outer-circle {
+  border: solid 2px;
+  border-radius: 50%;
+  box-sizing: border-box;
+  height: $radio-width;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transition: border-color ease 0.28s;
+  width: $radio-width;
+}
+
+.md-radio-inner-circle {
+  border-radius: 50%;
+  box-sizing: border-box;
+  height: $radio-width;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transition: transform ease 0.28s, background-color ease 0.28s;
+  transform: scale(0);
+  width: $radio-width;
+
+  .md-radio-checked & {
+    transform: scale(0.5);
+  }
+}
+
+.md-radio-label {
+  display: inline-block;
+  float: right;
```

Comment:
I was planning to hold off on RTL until a later PR, since there might need to be more thought put into it.